### PR TITLE
augeas: trim spaces before key value

### PR DESCRIPTION
**Pull request by mhabrnal**
Resolves: rhbz#1236613

Signed-off-by: Matej Habrnal <mhabrnal@redhat.com>

### CODE REVIEW

File: augeas/test_libreport.aug [hunk 1]
```diff
@@ -21,6 +21,10 @@ DontMatchComponents = selinux-policy
 # for more info about these settings see: https://github.com/abrt/abrt/wiki/FAQ#creating-private-bugzilla-tickets
 CreatePrivate= no
 PrivateGroups=private
+ Whitespace = start
+ Whitespace_two=start
```
Comment (jfilak) [inline augeas/test_libreport.aug]: Missing two spaces here.

**Comment by mhabrnal** [comment]
PR updated

File: augeas/test_libreport.aug [hunk 2]
```diff
@@ -21,6 +21,10 @@ DontMatchComponents = selinux-policy
 # for more info about these settings see: https://github.com/abrt/abrt/wiki/FAQ#creating-private-bugzilla-tickets
 CreatePrivate= no
 PrivateGroups=private
+ Whitespace = start
+  Whitespace_two=start
+	Whitespace_three=start
+	 Whitespace_four=start
 "
```
Comment (jfilak) [inline augeas/test_libreport.aug]: Make the cases more diverse, please.

**Comment by mhabrnal** [comment]
PR updated

**Pull request merged**